api/v1beta3/vcdcluster_types.go

```diff
@@ -160,9 +147,6 @@ type MultiZoneSpec struct {
 	// DCGroupConfig contains configuration for DCGroup zone topology.
 	// +optional
 	DCGroupConfig DCGroupConfig `json:"dcGroupConfig,omitempty"`
-	// UserSpecifiedEdgeGatewayConfig contains configuration for UserSpecifiedEdgeGateway zone topology.
-	// +optional
-	UserSpecifiedEdgeGatewayConfig UserSpecifiedEdgeGatewayConfig `json:"userSpecifiedEdgeGatewayConfig,omitempty"`
 	// ExternalLoadBalancerConfig contains configuration for ExternalLoadBalancer zone topology.
```

Comment:
We should remove this as well. The details are mentioned below in another comment.